labrad/types.py

```diff
@@ -1459,19 +1459,19 @@ def __repr__(self):
         return 'Error(code=%r, msg=%r, payload=%r)' % (self.code, self.msg, self.payload)
 
     def __lrtype__(self):
-        return LRError(getType(self.payload))
+        return TError(getType(self.payload))
 
     def __lrflatten__(self, endianness):
         s, t, _ = flatten((self.code, self.msg, self.payload), endianness=endianness)
-        return s, LRError(t.items[2])
+        return s, TError(t.items[2])
 
 class Int(int):
     def __lrtype__(self):
-        return LRInt()
+        return TInt()
 
 class Word(int):
```

Comment:
I don't know whether this is used anywhere, but might be good to rename it to `UInt`